pyci/cost_ci.py

```diff
@@ -0,0 +1,53 @@
+# This file is part of PyCI.
+#
+# PyCI is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your
+# option) any later version.
+#
+# PyCI is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+# for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PyCI. If not, see <http://www.gnu.org/licenses/>.
+
+r"""PyCI Griebel-Knapek CI module."""
+
+from . import pyci
+
+
+__all__ = [
+    "add_energy",
+]
+
+
+def add_energy(wfn, energies, t=-0.5, p=1.0):
```

Comment:
Following the previous comment, we can change a name to add_cost. And then provide a cost, instead of energies here as a parameter